docs/examples/UserGuide/iceloss_ex.jl

```diff
@@ -31,49 +33,47 @@ Z = [repeat([i],c) for (i, c) = enumerate(cnt)];
 Z = reduce(vcat,Z);
 
 ## make color map
-cmap1 = RGBf.(Makie.to_colormap(:thermal))
+cmap1 = RGBf.(Makie.to_colormap(:thermal));
 nc = length(cmap1);
 alpha0 = zeros(nc);
 cmap = RGBA.(cmap1, alpha0);
```

Comment:
From a comment on Discourse by Julius, you could also use eg:
```julia
Colors.alphacolor.(Makie.to_colormap(:thermal), alpha0)
```
to multiply the colormap by those alphas.  Same for when within the loop.